DomainContent/LoadTest/RaffleWheel/raffleWheel.js

```diff
@@ -0,0 +1,138 @@
+//
+// raffleWheel.js
+// 
+// Created by Rebecca Stankus on 07/05/2018
+// Copyright High Fidelity 2018
+//
+// Licensed under the Apache 2.0 License
+// See accompanying license file or http://apache.org/
+//
+/* global Avatar, AccountServices */
+
+(function() {
+    var ANGULAR_VELOCITY = {
+        x: 10,
+        y: 0,
+        z: 0
+    };
+    var ANGULAR_VELOCITY_CHECK_MS = 50;
+    var ANGULAR_VELOCITY_DECREMENT = 0.09;
+    var CHECKING_INTERVAL_DELAY = 100;
+    var USERS_ALLOWED_TO_SPIN_WHEEL = ['philip', 'ryan', 'Becky', 'MissLiviRose', 'Firebird25', 'Alan_'];
+    var MAX_CHAR_PER_LINE = 9;
+    var CUT_OFF_1 = 13;
+    var CUT_OFF_2 = 18;
+    var CUT_OFF_3 = 24;
+    var CUT_OFF_4 = 32;
+    var LINE_HEIGHT_1 =0.21;
+    var LINE_HEIGHT_2 =0.15;
+    var LINE_HEIGHT_3 =0.115;
+    var LINE_HEIGHT_4 =0.075;
+    var DEFAULT_LINE_HEIGHT = 0.3;
+
+
+    var _this;
+
+    function shuffle(array) {
+        var currentIndex = array.length, temporaryValue, randomIndex;
+      
+        while (0 !== currentIndex) {
```

Comment:
Generally you'll want to do this the other way around (currentIndex !== 0) per the coding guidelines